NEWS.md

```diff
@@ -47,70 +33,113 @@
   environment variable with the current value of that environment variable
   (#667).
 
-* `deployTFModel()` is defunct. Posit Connect no longer supports hosting of
+## Lifecycle changes
+
+* Non-libcurl `rsconnect.http` options have been deprecated. This allows us to 
+  focus our efforts on a single backend, rather than spreading development
+  efforts across five. The old backends will remain available for at least 2
+  years, but if you are using them because libcurl doesn't work for you, please
+  report the problem ASAP so we can fix it.
+
+* `addConnectServer()` has been deprecated because it does the same
+  thing as `addServer()`.
```

Comment:
`addServer` rewrites its incoming URL, which did not happen previously -- that change made these two functions identical.